src/PostgREST/SchemaCache.hs

```diff
@@ -1025,6 +1044,71 @@ allViewsKeyDependencies =
       having ncol = array_length(array_agg(row(col.attname, view_columns) order by pks_fks.ord), 1)
       |]
 
+initialMediaAggregates :: MediaAggregateMap
+initialMediaAggregates =
+  HM.insert (RelAnyElement, MediaType.MTAny            ) BuiltinAggJson $
+  HM.insert (RelAnyElement, MediaType.MTApplicationJSON) BuiltinAggJson $
+  HM.insert (RelAnyElement, MediaType.MTSingularJSON   ) BuiltinAggSingleJson $
+  HM.insert (RelAnyElement, MediaType.MTTextCSV        ) BuiltinAggCsv $
+  HM.insert (RelAnyElement, MediaType.MTGeoJSON        ) BuiltinAggGeoJson
+  HM.empty
```

Comment:
The implementation is similar to the idea on https://github.com/PostgREST/postgrest/pull/1582#issuecomment-1483019360